practiceFlake8.py

```diff
@@ -0,0 +1,2 @@
+print('Estoy bien')
```

Comment:
This file is not needed! The linters need to be configured for all the py files, not just for one. Please update your makefile so you can run your linters from a single command.